k8s/client.go

```diff
@@ -25,6 +31,16 @@ import (
 // each existing endpoint / ingress produces a single update.
 const bufferedWatcherDuration = time.Millisecond * 50
 
+var once sync.Once
+var conflictingIngressStatusUpdates prometheus.Counter
+
+func initMetrics() {
+	once.Do(func() {
+		conflictingIngressStatusUpdates = metrics.RegisterNewDefaultCounter("client", "conflicting_status_updates",
+			"Count of ingress status updates which resulted in a conflict, but which did not cause an error")
+	})
```

Comment:
Is it necessary to have a metric for this? I feel since this is just meta information anyways, this is not something we need to be too worried about.

Reply:
Yeah, I guess it's not that relevant.